lib/foreman_maintain/cli/report_command.rb

```diff
@@ -0,0 +1,25 @@
+module ForemanMaintain
+  module Cli
+    class ReportCommand < Base
+      extend Concerns::Finders
+
+      option '--output', 'FILE', 'Output the generate report into FILE'
+      subcommand 'generate', 'Generates the usage reports' do
```

Comment:
having just one subcommand feels unnatural, but I guess this is preparation to also have `upload` or something?

Reply:
yes, this is open ended atm but we would like to have some direct upload in near future